cypress/support/commands.js

```diff
@@ -0,0 +1,41 @@
+// ***********************************************
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+import { login } from './commands/login.js';
+import { PLANS, PASSWORD_MODE } from '../fixtures/accountTypes.js';
+const users = require('../fixtures/accounts.json');
+
+/**
+ * [Will perform login using one of the predefined test accounts.]
+ * @param  {String} [plan="free"] [The type of the account. Ex: free, visionary, etc.]
+ * @param  {String} [passwordMode="regular"] [The password mode. Can be: regular, 2FA, 2PasswordMode or 2FA2PasswordMode.]
+ */
+Cypress.Commands.add('autoLogin', (plan = PLANS.free, passwordMode = PASSWORD_MODE.regular) => {
+    if (plan === PLANS.free && passwordMode === PASSWORD_MODE.twoFactorTwoPasswordMode) {
+        return login({
+            username: users.free_twoFactor_twoPasswordMode.username,
+            password: users.free_twoFactor_twoPasswordMode.password,
+            OTP: users.free_twoFactor_twoPasswordMode.OTP,
+            secondPassword: users.free_twoFactor_twoPasswordMode.secondPassword
```

Comment:
It would be better to have FREE_TWOFACOR_TWOPASSWORD_MODE.secondPASSWORD imho.
Also use destructuring on users, it would be easier to read ;)

Reply:
I changed the object names to be uppercase. The object attributes should stay camelCase.